project/Release.scala

```diff
@@ -1,14 +1,12 @@
 import sbt._
 import Keys._
-import bintray.BintrayKeys._
 import sbtrelease.ReleasePlugin.autoImport._
 import sbtrelease.ReleaseStateTransformations._
 
 object Release {
 
   def releaseSettings = Seq(
-    crossSbtVersions := Vector("1.3.10", "0.13.16"),
-    organization := "org.allenai.plugins",
+    organization := "org.allenai",
```

Comment:
why this change?

Reply:
consistency. It doesn't functionally change anything, but all of our libraries are under the org.allenai organization.